doc-content/enterprise-only/ai/ai-con.adoc

```diff
@@ -10,7 +10,7 @@ When you think about artificial intelligence (AI), machine learning and big data
 * Mathematical optimization: The use of conditions and constraints to find the optimal solution to a problem
 * Digital decisioning: The use of defined criteria, conditions, and a series of machine and human tasks to make decisions
 
-//COMMENT: Jess, graphic from slide one, red building blocks (just the blocks, not the heading)
+image:ai/123_RHPAM_Integrating_AI_1020_building-blocks.svg[]
 
 While science fiction is filled with what is referred to as artificial general intelligence (AGI), machines that perform better than people and cannot be distinguished from them and learn and evolve without human intervention or control, AGI is decades away. Meanwhile, we have pragmatic AI which is a lot less frightening and much more useful to us today. Pragmatic AI is a collection of AI technologies that combined provide solutions to problems such as predicting customer behavior, providing automated customer service, and helping customers make purchasing decisions.
```

Comment:
I think that this needs a minor tweak "Pragmatic AI is a collection of AI technologies that combined provide"

Pragmatic AI is a collection of AI technologies, that when combined, provide

We can discuss if you'd like.

Reply:
Good one!